src/lib.rs

```diff
@@ -720,7 +774,7 @@ impl Host {
     }
 
     /// Returns the list of actors in the host that contain all of the tags in the
-    /// supplied parameter
+    /// supplied parameter. This function will not make a lattice-wide tag query
```

Comment:
Theres a similar comment on line 757-758, I don't see it yet but something to think about is if we get a decent amount of functions that differ in their functionality with lattice mode on / off, we may want to find a way to easily distinguish them so there's less ambiguity for the developer of the host

Reply:
I ultimately want to get rid of non-lattice mode, and aim for having lattice be the normal behavior, and we can simply choose in or out of process lattice.